docs/roman/tweakreg/tweakreg_examples.rst

```diff
@@ -3,16 +3,28 @@ Examples
 In the examples below, `img` is either a string with the filename of a Roman `ASDF` file
 or a Roman datamodel `ImageModel`.
 
-1. To run TweakReg on a python session one image file
-   (i.e. one Roman's SCA) with the default parameters:
+#. To run TweakReg in a Python session with the default parameters:
 
         .. code-block:: python
 
                 from romancal.tweakreg.tweakreg_step import TweakRegStep
                 step = TweakRegStep()
-                step.process([img])
+                step([img])
```

Comment:
We are advertising using `Step.call` because this guarantees using all parameters and avoids confusion. The examples should read

```
 output = TweakRegStep.call(...)
```

As written they use `Step.run` which instantiates a new step and ignores any parameters users pass.